### DISCUSSION TITLE
How to ignore the pyflake error like pyflake code "F403"

### DISCUSSION THREAD

**Discussion by OriLiMu** [LazyVim/LazyVim#2817] [Q&A] [answered]
I know it may related to lsp or lint things. But I don't know how to start. 

I want to ignore the error notification for pyflake code "F403"

**Comment by dpetka2001** [comment, marked as answer]
Easiest way is to define which rules should be ignored in your `pyproject.toml`, assuming that `ruff` is doing your linting.

**Reply by OriLiMu** [reply]
How to know current lint? 
The error msg shows it is the `pyflake`
But after I run `Mason` I cant find pyflake installed. Just `python-lsp-server`

![image](https://github.com/LazyVim/LazyVim/assets/11295355/0d9b975a-bd5a-4e74-a024-4a5e6035bd27)

**Reply by dpetka2001** [reply]
`python-lsp-server`, when installed via Mason, it installs all the providers mentioned [here](https://pypi.org/project/python-lsp-server/). So, it also includes `pyflakes`. You have to read the docs of it for how to disable the rules you want.
